crates/execution/src/trailing.rs

```diff
@@ -513,17 +338,18 @@ mod tests {
             .quantity(Quantity::from(1))
             .build();
 
-        let result = trailing_stop_calculate(Price::new(0.01, 2), &order, None, None, None);
+        let result =
+            trailing_stop_calculate(Price::new(0.01, 2), None, None, &order, None, None, None);
 
         // TODO: Basic error assert for now
         assert!(result.is_err());
     }
 
     #[rstest]
-    #[case(OrderSide::Buy, 100.0, 1.0, 99.0, None)] // Last price 99 > trigger 98, no update needed
```

Comment:
I think comments beside test cases can be helpful, we should leave these in.

Reply:
sorry, added back, told u this was rushed